README.md

```diff
@@ -10,19 +10,21 @@
 
 ### Fast pattern-matching library
 
-Quamina provides APIs to create an interface called 
-a **Matcher**,
-add multiple **Patterns** to it, and then query JSON blobs
-called **Events** to discover which of the patterns match 
+**Quamina** implements a data type that has APIs to 
+create an instance and add multiple **Patterns** to it, 
```

Comment:
question: along the text, there's a mix of "Patterns" vs "patterns". Is this intentional?

Reply:
I was trying to be principled and capitalize when I was using it to specifically mean the Quamina type.  But there's been a lot of editing and it's possible there are breakages.